tests/test_interaction.py

```diff
@@ -1010,3 +1011,42 @@ def test_slender_body_matrix_product_only_xy(self, n_elem):
         slender_body_theory.apply_forces(rod)
 
         assert_allclose(correct_forces, rod.external_forces, atol=Tolerance.atol())
+
```

Comment:
Lets remove this block. We shouldn't have test case checking deprecation error/warning message. I'll add a commit to ignore.

[b1fa5d4](https://github.com/Ali-7800/PyElastica/commit/b1fa5d406198115df09f84c5d5b50547b59a1b22)